src/common/reply.h

```diff
@@ -0,0 +1,13 @@
+#ifndef REPLY_H
+#define REPLY_H
+
+// Meta definitions
+#define RPL_META_MESSAGE(servername, numeric, message) (std::string(":") + servername + " " + numeric + " " + message + "\r\n")
+
+// Numeric replies in order
+#define RPL_WELCOME_001(servername, nick, user, host) (RPL_META_MESSAGE(servername, "001", "Welcome to the Internet Relay Network " + nick + "!" + user "@" + host))
+#define RPL_YOURHOST_002(servername, version) (RPL_META_MESSAGE(servername, "002", "Your host is " + servername + ", running version " + version))
```

Comment:
Here the IRCv3 spec calls for the message contents be prepended with the colon ```:```, while RFC 2812 does not have the colon.
Another difference is that in IRCv3, the first parameter should be the client nickname while RFC 2812 does not include the client nickname in the message contents.
(identical conflicts as with reply 001)

| spec. | message parameters |
| --- | --- |
| RFC 2812 | ```Your host is <servername>, running version <ver>``` |
| IRCv3 | ```<client> :Your host is <servername>, running version <version>``` |